lib/action_network_rest/signatures.rb

```diff
@@ -13,7 +13,16 @@ def create(signature_data, tags: [])
       end
 
       response = client.post_request base_path, post_body
-      object_from_response(response)
+      response_object = object_from_response(response)
+
+      if response_object[:_links].any? && !response_object[:_links][:'osdi:person'].nil?
```

Comment:
can we do `response_object.dig('_links', 'osdi:person').present?` Or does that only work in Rails?

Reply:
`present?` is defined on active_support (https://api.rubyonrails.org/classes/Object.html#method-i-present-3F) that's why I avoided using it here